migrations/env.py

```
@@ -0,0 +1,61 @@
+import asyncio
+from logging.config import fileConfig
+
+from sqlalchemy import pool
+from sqlalchemy.engine import engine_from_config
+from sqlmodel import SQLModel
+from alembic import context
+
+from app.core.config import settings
+from app.models import *  # 모든 모델 임포트
```

Ruff (0.8.2)

`from app.models import *` used; unable to detect undefined names

(F403)